sigstore/verify/policy.py

```diff
@@ -93,6 +112,41 @@ def verify(self, cert: Certificate) -> None:
             )
 
 
+class _SingleX509ExtPolicyDer(_SingleX509ExtPolicy):
```

Comment:
Let's call this V2 or something, since they're both technically DER and Fulcio refers to these as "V2":

```suggestion
class _SingleX509ExtPolicyV2(_SingleX509ExtPolicy):
```

Reply:
done!